mlflow/utils/req_parsing.py

```diff
@@ -0,0 +1,77 @@
+import os
```

Comment:
Can we rename this file `requirements_utils.py`?

Reply:
Sure!